cli/pom.xml

```diff
@@ -23,6 +23,7 @@
     </modules>
 
     <properties>
+        <!-- TODO: Get Argot to support Scala 2.12 or something, I don't even know -->
```

Comment:
This is where I left off on this story. The core module succeeds on the `build` goal, but the CLI modules do not because of this dependency. Unfortunately, this library is abandoned (in favor of Scopt):

http://software.clapper.org/argot/

We will either have to update Argot to support Scala 2.12, or replace our usage of it with Scopt (or some other supported command parser). The latter would be preferred.